lib/pbkit/pbkit.c

```diff
@@ -79,6 +79,8 @@ struct s_CtxDma
     DWORD               isGr;
 };
 
+static unsigned int pb_ColorFmt = NV097_SET_SURFACE_FORMAT_COLOR_LE_A8R8G8B8;
```

Comment:
This hardcodes the default mode to 32bpp.
Previously, `vm.bpp` was used to select 16bpp or 32bpp mode in `pb_init`.

However, we can keep 32bpp hardcoded, because pbkit didn't even work in 16pp mode before, so every pbkit program only handles 32bpp mode (at least it didn't before #140 - I'll have to check if it works now). If anyone had `vm.bpp` at 16, pbkit would have crashed.